fastq_filtrator.py

```diff
@@ -0,0 +1,64 @@
+def gc(read, gc_bounds=(0, 100)): #the function for GC filtering
+    seq = read[1].rstrip('\n') #geting rid of '\n' at the end of the string
+    C = seq.count('C') #counting of Cytosine in the read
+    G = seq.count('G') #counting of  Guanin in the read
+    CG = round((((int(C) + int(G)) / (len(seq) + 1)) * 100), 1) #counting of GC proportion
+    if type(gc_bounds) == tuple: #checking what type of the argument we have in conditions
+        if float(gc_bounds[0]) <= CG <= gc_bounds[1]: #checking of compliance
+            return True
+        else:
+            return False
```

Comment:
Упростим, \+ не совсем понятно, зачем делать `float`
```suggestion
        return gc_bounds[0] <= CG <= gc_bounds[1]: #checking of compliance
```